tests/tokens/test_ERC1155ownable.py

```diff
@@ -0,0 +1,474 @@
+import pytest
+import brownie
+from brownie import ZERO_ADDRESS, accounts
+
+# ERC1155 ownable, opensea compatible tests
+# @author Dr. Pixel (github: @Doc-Pixel)
+
+# constants - contract deployment
+CONTRACT_NAME = 'TEST 1155'
+CONTRACT_SYMBOL= 'T1155'
+CONTRACT_URI = 'https://mydomain.io/NFTdata/{id}'
```

Comment:
I don't think the way you are using the URI in the tests works here (you can't do format strings)

Reply:
I deliberately derailed the test, the URI is updated correctly.

The usage of {id} is an optional part of the metadata standard for URI in erc1155.

` ERC-1155 Metadata URI JSON Schema
This JSON schema is loosely based on the “ERC721 Metadata JSON Schema”, but includes optional formatting to allow for ID substitution by clients. If the string {id} exists in any JSON value, it MUST be replaced with the actual token ID, by all client software that follows this standard.`
from: https://eips.ethereum.org/EIPS/eip-1155

![image](https://user-images.githubusercontent.com/33023023/163702450-b4d52b9b-4f65-4dee-8a14-0e7b6a90db58.png)